core/src/test/java/org/frankframework/http/rest/ApiListenerServletTest.java

```diff
@@ -220,6 +220,10 @@ private MockHttpServletRequest doCreateRequest(String uriPattern, Methods method
 			request.setContent(content);
 		}
 
+		if (method == Methods.HEAD) {
+			request.setContent(null);
+		}
+
```

Comment:
This is a strange bit of code. We have first an `if` to check that method != GET then we set content. Then we check method == HEAD and then oops we remove the content again.

That can be combined.